db/migrate/20240103124622_user_hidden_items.rb

```diff
@@ -0,0 +1,10 @@
+class UserHiddenItems < ActiveRecord::Migration[7.1]
+  def change
+    create_table :user_hidden_items do |t|
+      t.references :user, null: false, foreign_key: true
+      t.string :user_hideable_type, null: false
+      t.bigint :user_hideable_id
```

Comment:
Imho si to trosku skomplikoval, lebo ak vsetko v lavom menu povysime na "filtre" tak nielen mozeme menit poziciu ako chcem ale aj ziskame moznosti na subscribe na filtre + vie si to user vsetko nastavit a vyriesime aj stitky aj filtre aj vsetko a este sa zbavime tohto polymorphic association peklicka.